tools/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     networks:
       - rexa-network
   rexa-db-webadmin:
-    image: adminer
+    image: adminer:4.2.5-standalone
```

Comment:
what is this ?

Reply:
It's a tools to connect to the database to see the info.
I used it to see if flyway worked well.

I update the docker image version